File: caddisfly-app/app/src/androidTest/java/org/akvo/CbtSuite.java
package org.akvo;

import org.akvo.caddisfly.instruction.CbtInstructions;
import org.akvo.caddisfly.test.CbtTest;
import org.junit.runner.RunWith;
import org.junit.runners.Suite;

@RunWith(Suite.class)
@Suite.SuiteClasses({CbtInstructions.class, CbtTest.class})
public class CbtSuite {
}

File: caddisfly-app/app/src/androidTest/java/org/akvo/CbtSuite.kt
package org.akvo

import org.akvo.caddisfly.instruction.CbtInstructions
import org.akvo.caddisfly.test.CbtTest
import org.junit.runners.Suite

// Uncomment @Runwith below to run this suite. This is commented because gradlew runs the test twice
//@RunWith(Suite::class)
@Suite.SuiteClasses(CbtInstructions::class, CbtTest::class)
class CbtSuite
